pkg/azurefile/controllerserver.go

```diff
@@ -529,7 +533,7 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		// not necessary for dynamic file share name creation since volumeID already contains volume name
 		uuid = volName
 	}
-	volumeID = fmt.Sprintf(volumeIDTemplate, resourceGroup, accountName, validFileShareName, diskName, uuid, secretNamespace)
+	volumeID = fmt.Sprintf(volumeIDTemplate, resourceGroup, accountName, validFileShareName, diskName, uuid, secretNamespace, subsID)
```

Comment:
current volumeid contains subsid, is it possible to not append subsid by default, we only append subsid when it's cross subs?

`volumeid="kubetest-uyg5f4ur#fb2e232232e1d4e799ac350#pvc-ac1d3f51-9ed4-43c6-9ef0-486d09f05e3f###kube-system#0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"`

Reply:
What about this? keep `volumeIDTemplate` unchanged, and then
```
if subsID != "" {
  volumeID = volumeID + "#" + subsID
} 
```

by this way, volumeID format is not changed if it's not cross subs scenario.

Reply:
That's right